Starcounter.Uniform.Tests/ViewModels/ItemMessagesTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using Starcounter.Uniform.FormItem;
+using Starcounter.Uniform.Generic.FormItem;
+using Starcounter.Uniform.ViewModels;
+using System;
+using System.Collections.Generic;
+
+namespace Starcounter.Uniform.Tests.ViewModels
+{
+    public class ItemMessagesTests
+    {
+        private FormItemMetadata _sut;
+        private List<string> _multipleProperties => new List<string>
+        {
+            "Property1",
+            "Property2",
+            "Property3"
+        };
+        private string _property => "Name";
```

Comment:
This is a property now, because of the `=>`. To make it a field (or a const), use `=`. Just sayin'

Reply:
I know, that was my idea, to have only properties there. Should I make them fields instead?

Reply:
I don't really care, it's not important. It was just strange to me and I thought that maybe you did it by accident - I usually use a property for non-private stuff to encapsulate, but fields for privates.